2022-04-18 18:00:57,389 INFO    MainThread:119500 [wandb_setup.py:_flush():71] setting env: {}
2022-04-18 18:00:57,404 INFO    MainThread:119500 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-18 18:00:57,405 INFO    MainThread:119500 [wandb_init.py:_log_setup():371] Logging user logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_180057-zeon2lg2/logs/debug.log
2022-04-18 18:00:57,406 INFO    MainThread:119500 [wandb_init.py:_log_setup():372] Logging internal logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_180057-zeon2lg2/logs/debug-internal.log
2022-04-18 18:00:57,407 INFO    MainThread:119500 [wandb_init.py:init():404] calling init triggers
2022-04-18 18:00:57,408 INFO    MainThread:119500 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-18 18:00:57,409 INFO    MainThread:119500 [wandb_init.py:init():460] starting backend
2022-04-18 18:00:57,410 INFO    MainThread:119500 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-18 18:00:57,452 INFO    MainThread:119500 [backend.py:ensure_launched():216] starting backend process...
2022-04-18 18:00:57,472 INFO    MainThread:119500 [backend.py:ensure_launched():221] started backend process with pid: 119563
2022-04-18 18:00:57,481 INFO    MainThread:119500 [wandb_init.py:init():469] backend started and connected
2022-04-18 18:00:57,490 INFO    MainThread:119500 [wandb_init.py:init():533] updated telemetry
2022-04-18 18:00:57,583 INFO    MainThread:119500 [wandb_init.py:init():563] communicating current version
2022-04-18 18:01:02,585 INFO    MainThread:119500 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-18 18:01:05,600 INFO    MainThread:119500 [wandb_init.py:init():606] starting run threads in backend
2022-04-18 18:01:10,628 INFO    MainThread:119500 [wandb_run.py:_console_start():1810] atexit reg
2022-04-18 18:01:10,632 INFO    MainThread:119500 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-18 18:01:10,633 INFO    MainThread:119500 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-18 18:01:10,639 INFO    MainThread:119500 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-18 18:01:10,640 INFO    MainThread:119500 [wandb_init.py:init():633] run started, returning control to user process
2022-04-18 18:05:23,113 INFO    MainThread:119500 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-18 18:05:23,119 INFO    MainThread:119500 [wandb_run.py:_restore():1752] restore
2022-04-18 18:05:25,991 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:05:26,388 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:05:26,730 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 18:05:26,870 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:05:26,974 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:05:27,078 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:05:27,182 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:05:27,286 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 18:05:27,394 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:27,499 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:27,603 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:27,707 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:27,811 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:27,918 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:28,665 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 18:05:29,200 INFO    MainThread:119500 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}
local_info {
}

2022-04-18 18:05:33,702 INFO    MainThread:119500 [wandb_run.py:_append_files():2180] logging synced files
